app/helpers/mortgage_calculator/stamp_duties_helper.rb

```diff
@@ -11,6 +11,12 @@ def second_home_threshold
       )
     end
 
+    def ftb_threshold
+      formatted_currency(
+        MortgageCalculator::StampDuty::FIRST_TIME_BUYER_THRESHOLD
```

Comment:
@margOnline There is a minor issue with an undefined constant. See the error details below.

```
Failures:

  1) MortgageCalculator::StampDutiesHelper#ftb_threshold returns the first time buyer threshold formatted with a currency sign
     Failure/Error: MortgageCalculator::StampDuty::FIRST_TIME_BUYER_THRESHOLD

     NameError:
       uninitialized constant MortgageCalculator::StampDuty::FIRST_TIME_BUYER_THRESHOLD
     # ./app/helpers/mortgage_calculator/stamp_duties_helper.rb:16:in `ftb_threshold'
     # ./spec/helpers/stamp_duties_helper_spec.rb:27:in `block (3 levels) in <module:MortgageCalculator>'

Finished in 0.12387 seconds (files took 2.59 seconds to load)
4 examples, 1 failure

Failed examples:

rspec ./spec/helpers/stamp_duties_helper_spec.rb:26 # MortgageCalculator::StampDutiesHelper#ftb_threshold returns the first time buyer threshold formatted with a currency sign
```

# Solution 1: Define missing constant
```
#in file: app/models/mortgage_calculator/stamp_duty.rb

FIRST_TIME_BUYER_THRESHOLD = 500_000
SECOND_HOME_THRESHOLD = 40000
```

You may want to make a small tweak to the other constant whilst you are making enhancements to the code. 

`SECOND_HOME_THRESHOLD = 40000` --> `SECOND_HOME_THRESHOLD = 40_000`

Reply:
I think if this branch gets rebased then this issue should be fixed. I've now merged my branch in

Reply:
All PRs should have all tests passing. It should be treated as independent, therefore, it should be in a good working order before being merged with any other branch, especially `master`.

Reply:
agree @Tr4pSt3R my bad, it was supposed to be a pr against will's branch